scripts/update_api_description.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+readonly THIS_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd | xargs realpath)"
+readonly ROOT_DIR="${THIS_DIR}/.."
+readonly OUT_DIR="${ROOT_DIR}/resources"
+readonly OUT_FILE="${OUT_DIR}/api.json"
+
+function main() {
+    if [[ $# -ne 0 ]] ; then
+        echo "USAGE: $0"
+        exit 1
+    fi
+
+    trap exit SIGINT
+
+    cd "${ROOT_DIR}"
+
+    mkdir -p "${OUT_DIR}"
+
+    go run cmd/describe-api/main.go > "${OUT_FILE}"
+
+    if [ $? -eq 0 ]; then
+        echo "API description successfully updated in api.json."
+    else
+        echo "Failed to update api.json."
+        return 1
```

Comment:
Since there is a return here and the conditional branches are not symmetric (they both don't return),
I would invert this conditional and only have one case (which returns).
Then the success case is just the default.